commit 74bc89d2d55cfbe6ad050898f86188f62cda06c8
Author: Twisted Playground <twistedumbrella@gmail.com>
Date:   Wed Dec 21 15:34:51 2011 -0500

    "Sexy Switch" Implementation
    
    "Sexy Switch" Implementation is an extension of the "qcom Legacy
    Implementation"

diff --git a/libs/ui/Android.mk b/libs/ui/Android.mk
index e4304e8..d2098c3 100644
--- a/libs/ui/Android.mk
+++ b/libs/ui/Android.mk
@@ -40,6 +40,23 @@ include $(BUILD_HOST_STATIC_LIBRARY)
 
 include $(CLEAR_VARS)
 
+ifneq ($(BOARD_USES_QCOM_LEGACY),true)
+
+LOCAL_SRC_FILES:= \
+	$(commonSources) \
+	EGLUtils.cpp \
+	FramebufferNativeWindow.cpp \
+	GraphicBuffer.cpp \
+	GraphicBufferAllocator.cpp \
+	GraphicBufferMapper.cpp \
+	GraphicLog.cpp \
+	InputTransport.cpp \
+	PixelFormat.cpp \
+	Rect.cpp \
+	Region.cpp
+
+else
+
 LOCAL_SRC_FILES:= \
 	$(commonSources) \
 	EGLUtils.cpp \
@@ -55,6 +72,8 @@ LOCAL_SRC_FILES:= \
 	Rect.cpp \
 	Region.cpp
 
+endif
+
 LOCAL_SHARED_LIBRARIES := \
 	libcutils \
 	libutils \
